### ISSUE TITLE
Use `<video autoplay>` attribute

### ISSUE THREAD

**Issue by antfu** [elk-zone/elk#1324]
ttps://www.w3schools.com/tags/att_video_autoplay.asp

**Comment by rshigg** [comment]
@niklaswolf @antfu 
PR for this here: https://github.com/elk-zone/elk/pull/1325

**Comment by ZMYaro** [comment]
Just to be sure, is the browser default autoplay experience the experience we want?  I know Twitter specifically waits until the video scrolls into view before it starts playing, which I do not think just adding the `autoplay` attribute does.  (Please correct me if I am wrong and there is a JavaScript-free way to achieve that!)

**Comment by niklaswolf** [comment]
@ZMYaro you're right in my view. The autoplay attribute does auto-play the video when it's ready, disregarding if the element is actually visible in the viewport. But this (and that's how it's implemented currently) should be the desired behaviour. So I think the `autoplay` attribute does not give the desired experience, like you said
